### ISSUE TITLE
Errors on generating Javadoc with Maven and AdoptOpenJDK

### ISSUE THREAD

**Issue by heidivanparys** [opengeospatial/ets-ogcapi-features10#105]
I tried to build the code (mvn package) so I could use the "all-in-one" JAR file to run the tests locally but I got the following errors:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:03 min
[INFO] Finished at: 2020-03-26T21:50:48+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.7.1:jar (site-package) on project ets-ogcapi-features10: Error generating maven-javadoc-plugin:2.10.4:javadoc report:
[ERROR] Exit code: 1 - javadoc: error - The code being documented uses modules but the packages defined in http://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.
[ERROR] C:\Users\B004114\Documents\GitHub\heidivanparys\ets-ogcapi-features10\src\main\java\org\opengis\cite\ogcapifeatures10\util\JsonUtils.java:215: warning: no @param for rels
[ERROR]     public static List<String> findLinksWithoutRelOrType( List<Map<String, Object>> links, Set<String> rels ) {
[ERROR]                                ^
[ERROR]
[ERROR] Command line was: C:\OtherPrograms\AdoptOpenJDK\x64_11.0.2_9\jdk-11.0.2+9\bin\javadoc.exe @options @packages
[ERROR]
[ERROR] Refer to the generated Javadoc files in 'C:\Users\B004114\Documents\GitHub\heidivanparys\ets-ogcapi-features10\target\site\apidocs' dir.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

Updating the javadoc for method `JsonUtils.findLinksWithoutRelOrType()` resolves one issue, but the build still fails:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:46 min
[INFO] Finished at: 2020-03-26T21:55:32+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar (attach-javadocs) on project ets-ogcapi-features10: MavenReportException: Error while generating Javadoc:
[ERROR] Exit code: 1 - javadoc: error - The code being documented uses modules but the packages defined in http://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.
[ERROR]
[ERROR] Command line was: C:\OtherPrograms\AdoptOpenJDK\x64_11.0.2_9\jdk-11.0.2+9\bin\javadoc.exe @options @packages
[ERROR]
[ERROR] Refer to the generated Javadoc files in 'C:\Users\B004114\Documents\GitHub\heidivanparys\ets-ogcapi-features10\target\apidocs' dir.
[ERROR]
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**Comment by dstenger** [comment]
Thank you for reporting.
@keshav-nangare Can you please take a look at this problem?

**Comment by keshav-nangare** [comment]
@dstenger 

I went through the log, looks like you trying to build project with Java 11. There is an issue with javadoc in Java-11 reference to this issue https://bugs.openjdk.java.net/browse/JDK-8212233#descriptionmodule_heading

This issue is not occurring when we use Java 8 for a build. The following solution will fix this issue:

Update maven-javadoc-plugin with the following configuration

```
<configuration>
     <source>1.8</source>
</configuration>
```

**Comment by heidivanparys** [comment]
Thanks for sorting this out.

I checked the documentation for the [maven-javadoc-plugin](https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html), and the source can also be set using a user property.

![image](https://user-images.githubusercontent.com/11427611/77890865-684cf900-7270-11ea-9048-20e784176ebf.png)

So instead of updating pom.xml, the code can be built using `mvn package -Dsource=8`. Could this workaround be added to how-to-run-the-tests.adoc?

Note: `mvn package -Dsource=1.8` seems to work as well, although the use of "1.8" is not documented in the `javac --help` output:

```
[...]
--release <release>
        Compile for a specific VM version. Supported targets: 6, 7, 8, 9, 10, 11
  -s <directory>               Specify where to place generated source files
  -source <release>
        Provide source compatibility with specified release
[...]
```

**Comment by dstenger** [comment]
Nevertheless, we can also update the pom file.
@keshav-nangare Can you please create a pull request?

**Comment by keshav-nangare** [comment]
Fixed with #108 PR.
